federator/transferTest.js

```diff
@@ -77,16 +126,16 @@ async function run(federators) {
         logger.debug('Get the side token address');
         let sideBridgeContract = new sidechainWeb3.eth.Contract(abiBridge, config.sidechain.bridge);
         let sideTokenAddress = await sideBridgeContract.methods.mappedTokens(mainTokenAddress).call();
-        logger.info('Side token address', sideTokenAddress);
+        logger.info(`${destination} token address`, sideTokenAddress);
 
         logger.debug('Check balance on the other side');
         let sideTokenContract = new sidechainWeb3.eth.Contract(abiMainToken, sideTokenAddress);
```

Comment:
same here, i prefer that
sideTokenContract to be  destinationTokenContract
sideBridgeContract to be destinationBridgeContract
sideTokenAddress to be destinationTokenAddress

I think it brings a bit of clarity now that we use the same function for moving tokens from both networks for this test